suite-native/module-accounts-import/src/components/AccountImportHeader.tsx

```diff
@@ -10,19 +23,51 @@ const accountImportHeaderStyle = prepareNativeStyle(_ => ({
     width: '100%',
 }));
 
+const closeButtonSizeStyle = prepareNativeStyle(_ => ({
+    width: 48,
+    height: 48,
+}));
+
+type NavigationProp = StackToTabCompositeProps<
+    AccountsImportStackParamList,
+    AccountsImportStackRoutes.AccountImport,
+    RootStackParamList
+>;
+
 export const AccountImportHeader = ({ activeStep }: AccountImportHeaderProps) => {
     const { applyStyle } = useNativeStyles();
+    const navigation = useNavigation<NavigationProp>();
+    const isOnboardingFinished = useSelector(selectIsOnboardingFinished);
+
+    const handleCloseOnboarding = () =>
+        navigation.navigate(RootStackRoutes.AppTabs, {
+            screen: AppTabsRoutes.HomeStack,
+            params: {
+                screen: HomeStackRoutes.Home,
+            },
+        });
+
     return (
         <Box
             flexDirection="row"
-            justifyContent="center"
+            justifyContent={isOnboardingFinished ? 'space-between' : 'center'}
             alignItems="center"
             marginBottom="small"
             style={applyStyle(accountImportHeaderStyle)}
```

Comment:
Please avoid "inline" styling and normal styling using `style` property. If you use style property always move all styles here.

Reply:
move it to style obj